### DISCUSSION TITLE
User's history view

### DISCUSSION THREAD

**Discussion by marco-qg** [Arquisoft/wiq_es6c#49] [Polls]
I'm not sure about this view's appearance. What do you think? Feel free to share new ideas too!

![Captura de pantalla 2024-03-07 200432](https://github.com/Arquisoft/wiq_es6c/assets/145330494/9ab72bf3-4e3f-4020-8bd8-9162c08c3d34)
![Captura de pantalla 2024-03-07 200503](https://github.com/Arquisoft/wiq_es6c/assets/145330494/47f13bd4-cc46-43b8-aa1e-9688c97c6282)

**Comment by AbelMH1** [comment]
I think that mixing both views could be the best! I would show two columns, but it would not be a column for each game, but two columns for the questions of the same game.

**Reply by marco-qg** [reply]
Great idea! Thank you for sharing it with us @AbelMH1!
